apstra/two_stage_l3_clos_client.go

```diff
@@ -797,11 +798,16 @@ func (o *TwoStageL3ClosClient) GetIbaProbe(ctx context.Context, id ObjectId) (*I
 	return probe, err
 }
 
-// DeleteIbaProbe deletes an IBA Widget
+// DeleteIbaProbe deletes an IBA Probe
 func (o *TwoStageL3ClosClient) DeleteIbaProbe(ctx context.Context, id ObjectId) error {
 	return o.client.deleteIbaProbe(ctx, o.blueprintId, id)
 }
 
+// CreateIbaProbe creates an IBA Probe
+func (o *TwoStageL3ClosClient) CreateIbaProbe(ctx context.Context, probeJson json.RawMessage) (ObjectId, error) {
```

Comment:
It's possible we'll flesh out support for IBA probes in the future. These functions will have taken the obvious names for those future capabilities.

Should these new public and private methods be named with `FromJson` instead, so that the usual methods can be slotted in later?

This is probably an argument for renaming `GetIbaProbe` as well (maybe a good time for it if you're working on the only consumer anyway).

Reply:
new struct added and new function created